code/datums/ammo/bullet/bullet.dm

```diff
@@ -47,6 +47,8 @@
 		SPAN_HIGHDANGER("[user] aims \the [fired_from] directly at your head!"),
 		SPAN_DANGER("[user] aims \the [fired_from] at [execution_target]'s head!"))
 
+	user.emote("me", 1, "aims [fired_from] at [execution_target]'s head!")
```

Comment:
i don't think m_type, and thereby /run_emote(... type_override ...) is even used, why is it 1 here?

Reply:
I honestly couldn’t figure it out after a while of trying and forgot about it, the few emote()’s I saw used it so I assumed it was important, I can mass fix it